src/control/lib/control/pool_property.go

```diff
@@ -116,6 +123,50 @@ func PoolProperties() PoolPropertyMap {
 				jsonNumeric: true,
 			},
 		},
+		"ec_pda": {
+			Property: PoolProperty{
+				Number:      drpc.PoolPropertyECPda,
+				Description: "Performance domain affinity level of EC",
+				valueHandler: func(s string) (*PoolPropertyValue, error) {
+					ecpdaErr := errors.Errorf("invalid ec_pda value %s", s)
+					pdaPct, err := strconv.ParseUint(s, 10, 64)
+					if err != nil || !C.daos_ec_pda_valid(C.uint32_t(pdaPct)) {
+						return nil, ecpdaErr
+					}
+					return &PoolPropertyValue{pdaPct}, nil
+				},
+				valueStringer: func(v *PoolPropertyValue) string {
+					n, err := v.GetNumber()
+					if err != nil {
+						return "not set"
+					}
+					return fmt.Sprintf("%d", n)
+				},
+				jsonNumeric: true,
+			},
+		},
+		"rp_pda": {
+			Property: PoolProperty{
+				Number:      drpc.PoolPropertyRPPda,
+				Description: "Performance domain affinity level of RP",
+				valueHandler: func(s string) (*PoolPropertyValue, error) {
+					rppdaErr := errors.Errorf("invalid rp_pda value %s", s)
+					pdaPct, err := strconv.ParseUint(strings.ReplaceAll(s, "%", ""), 10, 64)
+					if err != nil || !C.daos_ec_pda_valid(C.uint32_t(pdaPct)) {
```

Comment:
I am a little confused by this... Is this a domain affinity level or a percentage? If it's really supposed to be a percentage, then a percentage of what? Maybe the description string needs to be more verbose for these properties. A user should be able to figure out what they are for without having to look at separate documentation.

Reply:
Sorry, this is a copy-paste problem, it should be a value, not percentage, i will remove replaceAll()